pkg/controller/account/iam.go

```diff
@@ -174,7 +177,7 @@ func (r *ReconcileAccount) BuildSTSUser(reqLogger logr.Logger, awsSetupClient aw
 	// with eventual consisency on AWS' side
 	time.Sleep(10 * time.Second)
 
-	// Create STS user for SRE admins
+	// Create the sre-admin user credentials
```

Comment:
This is specifically STS credentials, which is different from the normal Access Keys